README.md

```diff
@@ -96,9 +95,11 @@ You will need to have an [AWS account already created](https://aws.amazon.com)
 #### Note: To Deploy the national parks application with EAS Beta you must follow these additional instructions:
```

Comment:
What variables do we need for national-parks + Automate? 

I see this listed 
`edit terraform.tfvars with your own values and add in the automate_url and automate_token from the previous step`

We mentioned event-stream-enabled, and 'hab-sup-version' 

I also set automate_hostname, and automate_ip based on the variables in the tfvars.example. This may not be anything added with the dashboard MVP but something we missed when we added automate.

Reply:
> Lots of changes, does it make sense for folks who haven't dug into the details of the EAS beta yet?

I think they're prime candidates. Removes any expertise bias from the equation, since folks that have done it manually are more likely to take settings for granted. :-)

Reply:
Good call. I didn't point them out in the readme, but put them in the tfvars example.
the `automate_ip` and `automate_token` are the requisite pieces for EAS. I believe `automate_hostname` and/or `automate_url` are used in the infra/inspec data collection bits, so they were in there before I started. No idea what the `automate_user` var is used for off hand.

Reply:
Updated in the README to explicitly instruct folks to make sure `automate_ip` and `automate_token` are set.